linked_list/linked_list.py

```diff
@@ -21,8 +21,13 @@ class LinkedList:
   def __init__(self):
     self.head = None
     self.tail = None
-
+    self.list = None
```

Comment:
Ah, I see what you were asking now. You should not have to add `self.list` here.